.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: Update terraform dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 6 * * *'
+
+jobs:
+  dependabot-terraform:
+    runs-on: ubuntu-latest
+    steps:
+      - name: update terraform dependencies
+        uses: plus3it/dependabot-terraform-action@p3-implementation
+        with:
+          github_dependency_token: ${{ secrets.GH_READONLY_TOKEN }}
+          directory: |
```

Comment:
Do you need the "directory"?  Isn't that redundant given what's in the `.dependabot.yml` file?

Reply:
It would certainly be nice if it were redundant, but unfortunately the github action uses its own config and does not read the dependabot.yml file.